api/views/submission.py

```diff
@@ -778,3 +778,99 @@ def bulkcheck(self, request: Request) -> Response:
                 urls.pop(urls.index(submission.url))
 
         return Response(status=status.HTTP_200_OK, data=urls)
+
+    @csrf_exempt
+    @swagger_auto_schema(
+        manual_parameters=[
+            Parameter(
+                "user_id",
+                "query",
+                type="number",
+                description="The user to center the leaderboard on.",
+            ),
+            Parameter(
+                "top_count",
+                "query",
+                type="number",
+                description="The number of users to show from the top leaderboard.",
+            ),
+            Parameter(
+                "above_count",
+                "query",
+                type="number",
+                description="The number of users to show above the given user.",
+            ),
+            Parameter(
+                "below_count",
+                "query",
+                type="number",
+                description="The number of users to show below the given user.",
+            ),
+        ],
+        responses={404: "No volunteer with the specified ID."},
+    )
+    @action(detail=False, methods=["get"])
+    def leaderboard(self, request: Request,) -> Response:
+        """Get the leaderboard for the given user."""
+        user_id = request.GET.get("user_id", None)
+        if user_id is not None:
```

Comment:
Can just shorten to `if user_id`

Reply:
Would that still work if `user_id` is `0`? Or does that evaluate to `false`?
In JS I always like to do it explicitly to avoid these kind of errors, not sure how Python works in that regard

Reply:
I think 0 in Python is Falsy, so that wouldn't work.

Reply:
In that case I'll probably leave it like this, just to be sure

Reply:
We can keep it as is; however, user IDs start from 1, so there's no reason why the user ID would be zero. Either way, works for me.